Gems/WarehouseAssets/Assets/Prefabs/Warehouse_structural/Components/Ceeling_Lamp.prefab

```diff
@@ -0,0 +1,374 @@
+{
+    "ContainerEntity": {
+        "Id": "ContainerEntity",
+        "Name": "Ceeling_Lamp",
```

Comment:
nit: This is super minor but I think this should be `ceiling` lamp.

Reply:
Fixed typo.

Reply:
Thanks!